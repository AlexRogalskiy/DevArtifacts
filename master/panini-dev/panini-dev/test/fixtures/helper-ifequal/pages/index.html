---
kittens: kittens
---
{{#ifequal kittens 'kittens'}}
Yes, kittens.
{{/ifequal}}

{{#ifequal kittens 'notkittens'}}
Yes, kittens.
{{else}}
Not kittens.
{{/ifequal}}
