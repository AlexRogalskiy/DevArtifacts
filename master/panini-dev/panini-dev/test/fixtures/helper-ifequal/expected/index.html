<html>
  <body>
    Yes, kittens.
    
    Not kittens.
    
  </body>
</html>
